### PR TITLE
zkevm deployment

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by smartdevcom — APPROVED** [review]

File: packages/chains/deployments/zkevm.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "zkevm",
+  "chainId": "1101",
+  "contracts": {}
```
Comment (vminkov) [inline packages/chains/deployments/zkevm.json:4]: where are the deployed contracts addresses/files? I saw the AddressesProvider is deployed at 0x27aA55A3D55959261e119d75256aadAB79aE897C

File: packages/chains/src/zkevm/params.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const specificParams: ChainParams = {
+  blocksPerYear: BigNumber.from((40 * 24 * 365 * 60).toString()),
+  cgId: "ethereum",
+  metadata: {
+    chainIdHex: "0x44D",
+    name: "Polygon zkEVM",
+    shortName: "zkEVM",
+    uniswapV3Fees: {},
+    img: "https://d1912tcoux65lj.cloudfront.net/network/polygon.png",
```
Comment (vminkov) [inline packages/chains/src/zkevm/params.ts:15]: the img is not loading in my browser

File: packages/chains/src/zkevm/params.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    blockExplorerUrls: { default: { name: "lineascan", url: "https://zkevm.polygonscan.com" } },
```
Comment (vminkov) [inline packages/chains/src/zkevm/params.ts:16]: name lineascan?

File: packages/sdk/chainDeploy/mainnets/zkevm.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  nativeTokenName: "Ethereum",
+  nativeTokenSymbol: "ETH",
+  stableToken: zkevm.chainAddresses.STABLE_TOKEN,
+  wBTCToken: zkevm.chainAddresses.W_BTC_TOKEN,
+  blocksPerYear: zkevm.specificParams.blocksPerYear.toNumber(),
+  uniswap: {
+    hardcoded: [],
+    uniswapData: [],
+    pairInitHashCode: ethers.utils.hexlify("0x1f475d88284b09799561ca05d87dc757c1ff4a9f48983cdb84d1dd6e209d3ae2"),
+    uniswapV2RouterAddress: "0xbdFa4a05372a10172EeEB75075c85FCbff521625",
```
Comment (vminkov) [inline packages/sdk/chainDeploy/mainnets/zkevm.ts:27]: 0xbdFa4a05372a10172EeEB75075c85FCbff521625 not a contract at zkevmscan

File: packages/sdk/chainDeploy/mainnets/zkevm.ts [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    uniswapV2FactoryAddress: "",
+    uniswapV3FactoryAddress: "0x865412B6cDf424bE36088fE3DeC2A072a26Cc494",
```
Comment (vminkov) [inline packages/sdk/chainDeploy/mainnets/zkevm.ts:29]: 0x865412B6cDf424bE36088fE3DeC2A072a26Cc494 either

**Review by vminkov — CHANGES_REQUESTED** [review]

File: packages/sdk/chainDeploy/mainnets/zkevm.ts (hunk 5 above)
Comment (carlomazzaferro) [reply packages/sdk/chainDeploy/mainnets/zkevm.ts:36]: yeah, there is actually neither deployed. Will set them to address zero

**Pull request merged**